py/desitarget/sv2/sv2_cuts.py

```diff
@@ -1475,7 +1475,7 @@ def isQSO_randomforest(gflux=None, rflux=None, zflux=None, maskbits=None,
 
     # Preselection to speed up the process
     rMax = 23.0   # r < 23.0
-    rMin = 16.5   # r > 16.5
+    rMin = 15.5   # r > 17.5
```

Comment:
Can you restore this to `rMin = 16.5`? We likely won't run any new SV2 target files, so we shouldn't change anything in `sv2_cuts.py`, if possible.